app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# Create Post model
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_counter
+  def update_posts_counter
+    author.increment!(:posts_counter)
```

Comment:
- [ ] Since the author does not exist in your database schema, this line should be updated after correcting the association between Post and User models.